packages/core/story-structure.js

```diff
@@ -57,10 +52,18 @@ const s = [
               'c/Datagrid/Extensions/Slug',
             ],
           },
+          { n: 'DataSpreadsheet', s: ['c/DataSpreadsheet'] },
           { n: 'TruncatedList', s: ['c/TruncatedList'] },
           { n: 'DelimitedList', s: ['c/DelimitedList'] },
+          { n: 'Modified tabs', s: ['c/ModifiedTabs'] },
+          { n: 'Options tile', s: ['c/OptionsTile'] },
+          { n: 'Page header', s: ['c/PageHeader'] },
+          { n: 'Side panel', s: ['c/SidePanel'] },
+          { n: 'Tag set', s: ['c/TagSet'] },
+          { n: 'Tearsheet', s: ['c/Tearsheet', 'c/TearsheetNarrow'] },
           { n: 'String Formatter', s: ['c/StringFormatter'] },
           { n: 'User avatar', s: ['c/UserAvatar'] },
+
```

Comment:
I wonder if this little space is why you got that merge conflict. 😅

Reply:
Let’s fix string formatter to sentence case and alphabetical since I think `String formatter` should come before `Tag set` anyway.